packages/xen-api/examples/utils.js

```diff
@@ -55,7 +55,7 @@ exports.pipeline = (...streams) => {
 }
 
 const resolveRef = (xapi, type, refOrUuidOrNameLabel) =>
-  isOpaqueRef(refOrUuidOrNameLabel)
+  Ref.is(refOrUuidOrNameLabel)
```

Comment:
Maybe `Ref.isRef`? because the method `is` doesn't exist in the file `_Ref`.

Reply:
What do you prefer?

I feel `Ref.isRef` is a repetition and it's not symmetric with `Ref.isNotEmpty`  but `Ref.is` appear a bit short :thinking:

Reply:
The two names are comprehensibles, but go for `Ref.is`.